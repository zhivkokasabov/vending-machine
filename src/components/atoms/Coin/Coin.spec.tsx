import { render } from "@testing-library/react";
import { Coin } from ".";

describe("Coin tests", () => {
  const renderComponent = () => render(
    <Coin denomination="1" className="className" />
  );

  it("should render", () => {
    const { asFragment } = renderComponent();

    expect(asFragment()).toMatchSnapshot();
  });
});
