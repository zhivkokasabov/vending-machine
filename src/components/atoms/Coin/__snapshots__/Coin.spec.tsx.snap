// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Coin tests should render 1`] = `
<DocumentFragment>
  <div
    class="flex items-center bg-gray-500 border-black text-white rounded-full justify-center border w-[3rem] h-[3rem] className"
    style="transform: perspective(500px) rotateX(50deg);"
  >
    1
  </div>
</DocumentFragment>
`;
